experimental/traffic-portal/src/environments/environment.ts

```diff
@@ -26,5 +26,6 @@ export const environment: Environment = {
 	apiVersion: "4.0",
 	customModule: false,
 	production: false,
-	useExhaustiveDates: true
+	tpv1Url: "https://tp.staging.cdnlab.comcast.net/#!",
```

Comment:
This file is a _compile_ time configuration setting. We need this redirect to be a _runtime_ configuration setting since the link should be able to change on the fly. This should probably go in the config file read from `server.ts`, and passed to the app through `TransferState` using ng universal.

Reply:
ok steve